code/modules/uplink/uplink_item.dm

```diff
@@ -1313,6 +1313,13 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	limited_stock = 2 //you can't use more than two!
 	restricted_roles = list("Shaft Miner")
 
+/datum/uplink_item/role_restricted/traitorsec
+	name = "Oppressor Upgrade Module"
+	desc = "A box containing an cyborg upgrade module that instantly hacks a cyborg and unlocks the outlawed securiy module (in a inconspicuous black-and-red paintjob)"
```

Comment:
securiy

Reply:
Also it's not explicitly outlawed, turning them off is a config setting. 

https://github.com/tgstation/tgstation/blob/6401bb8264708d3fc55c22fd6f7ba261aebc11ee/code/controllers/configuration.dm#L142
https://github.com/tgstation/tgstation/blob/7567fbafc43fa6430f165e6c702e5db5907d5733/code/modules/mob/living/silicon/robot/robot.dm#L212